### ISSUE TITLE
Please add documentation for #10945 (Custom OID polling and graphing)

### ISSUE THREAD

**Issue by CKeckNorthIO** [librenms/librenms#16870]
### The problem

For the relativly new feature "Add custom OID polling and graphing" #10945  there is no documentation available at https://docs.librenms.org/. It would be great to have at least a basic "howto", as the fields in the add-form are not all self-explaining. Thank you.

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.11.0-97-g8ba366fcf (2024-12-11T21:20:39+01:00)
DB Schema | 2024_10_20_154356_create_qos_table (310)
PHP       | 8.3.14
Python    | 3.10.12
Database  | MariaDB 10.6.18-MariaDB-0ubuntu0.22.04.1
RRDTool   | 1.7.2
SNMP      | 5.9.1
===========================================

[OK]    Composer Version: 2.8.3
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
```

### What was the last working version of LibreNMS?

24.11.0-97

**Comment by PipoCanaja** [comment]
@CKeckNorthIO This feature is not recommended at all anyway. This is not the standard way of monitoring OIDs in LibreNMS and it was multiple times asked if it would be kept or not. 
But feel free to open a PR to fix the documentation.

**Comment by CKeckNorthIO** [comment]
If it's "not recommended" - what then IS the recommended way to add custom OIDs as a check into LibreNMS? Making a PR for a full device-support is a bit overshoot for my taste if I only want to add a _single_ SNMP-based check.
I wish there was an officially supported way in LibreNMS for adding these single, custom SNMP-checks like it is possible with PRTG's "SNMP Custom Sensor" or "SNMP Custom Advanced Sensor".

**Comment by PipoCanaja** [comment]
Exactly, adding (or extending the already existing) support is the preferred way -> scales better, and most importantly, all users will benefit of it. A custom OID is per-device. So even you will have to do it multiple times if you have multiple identical devices, and every other LibreNMS user will have to reinvent the wheel.

**Comment by PipoCanaja** [comment]
It is not removed so far, clearly, but we don't push to make it very visible because it must remain the exception. 

If everybody uses custom-oids, then nobody contribute support for devices, and LibreNMS ends up with less devices supported, making it useless for all. 

Adding an entry to the YAML file is basically easier most of the time than dealing with CustomOIDs, so why would we push CustomOIDs anyway ?

**Comment by murrant** [comment]
Closing.  My preferred solution is to make a webui based option for augmenting yaml discovery data (and potentially a single click to submit that change upstream).

**Comment by CKeckNorthIO** [comment]
Thanks @murrant for the update. If a webUI for maintaining the yaml-files will come, I'm fine with that solution.
